lib/ecto/type.ex

```diff
@@ -400,6 +412,10 @@ defmodule Ecto.Type do
     end
   end
 
+  def dump({:parameterized, type, opts}, value) do
+    type.dump(value, opts)
+  end
```

Comment:
Should not be this written with `dumper` as 3rd argument like in clause with `{:embed, _}` match?

Reply:
I added dump/3 as another callback on ParameterizedType